api/v1alpha1/tlsconfig_types.go

```diff
@@ -0,0 +1,80 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+const (
+	// TLS1_0 denotes the TLS v1.0.
+	TLS1_0 = "TLS1_0"
+	// TLS1_1 denotes the TLS v1.0.
+	TLS1_1 = "TLS1_1"
+	// TLS1_2 denotes the TLS v1.0.
+	TLS1_2 = "TLS1_2"
+	// TLS1_3 denotes the TLS v1.0.
```

Comment:
copy pasta error. The godoc comments need to be updated.

Reply:
https://github.com/kubernetes-sigs/service-apis/blob/master/api/v1alpha1/gateway_types.go#L162-L171 has this godocs bug. I'll fix it and push a new commit.